### ISSUE TITLE
[HOLD][$250] Color: Many Pages: The links are identified only by use of color

### ISSUE THREAD

**Issue by elisenwazuoke** [Expensify/App#77244]
## Action Performed:
Prerequisites:
- the user is not signed in

1. Open the Expensify app
2. Note how the 'Licenses' link is displayed in the footer.

Other Occurrences:
1. On Sign In - Enter Code: 'Terms of Service', 'Privacy', 'Didn't receive a magic code?', 'Go back'
2. On Sign Up: 'Terms of Service', 'Privacy', 'Go back'
3. On Onboarding: the email, 'fix the errors'
4. On Profile - Contact Methods: the email address
5. On Profile - Display Name: 'fix the errors'
6. On Profile - Address: 'fix the errors'
7. On Profile - Legal Name: 'fix the errors'
8. On Security: 'Learn more'
9. On Security - Close Account: 'fix the errors'
10. On About: 'Terms of Service' and 'Privacy'
11. On Wallet - Add Bank Account: 'fix the errors'
12. On Save the World - I know a teacher: 'fix the errors'
13. On Troubleshoot: 'submit a bug'
14. On Workspace Details

## Expected Result:
The 'Licenses' link should be displayed with a clear, non-color visual cue, such as a text underline, to ensure all users recognize it as an interactive link.

## Actual Result:
The 'Licenses' text appears visually indistinguishable from surrounding plain text, with only a slight blue difference indicating its functionality. Users relying on non-color cues are unaware that the text is clickable, hindering navigation.

## Area issue was found in:
Sign in

## Failed WCAG checkpoints
WCAG 1.4.1 Use of Color
https://www.w3.org/WAI/WCAG22/Understanding/use-of-color

## User impact:
Users with partial sight often experience limited color vision. Some older users may not be able to see color well. Users who have color-blindness benefit when information conveyed by color is available in other visual ways. People using limited color monochrome displays may be unable to access color-dependent information. Users who have problems distinguishing between colors can look or listen for text cues.

## Suggested resolution:
Provide additional visual cues besides the link text color, e.g.: a permanent underline, bold or larger font, so that the button/link can be differentiated from other elements or surrounding text.
https://www.w3.org/WAI/WCAG21/Understanding/use-of-color.html

## Environment(s)
Samsung Galaxy S24 Ultra - Android 14, Partner - Native (No mobile browser)
Apple iPhone 11 - iOS 17.5.1, H0T Mobile - Native (No mobile browser)

## Screenshots/Videos
![Image](https://github.com/user-attachments/assets/2ae7acaf-c2c2-4f6c-aa3c-b1001f66de49)

**Version Number:** Production
**Reproducible in staging?:** NA
**Reproducible in production?:** Yes
**Email or phone of affected tester (no customers):** NA
**Logs:** https://stackoverflow.com/c/expensify/questions/4856 NA
**Notes/Photos/Videos:** Any additional supporting documentation Yes
**Expensify/Expensify Issue URL:** https://platform.applause.com/company/12102/products/36628/community-issues/7004754
**Issue reported by:** NA
**Slack conversation:** NA
<details><summary>Upwork Automation - Do Not Edit</summary>
    <ul>
        <li>Upwork Job URL: https://www.upwork.com/jobs/~022009685381211068045</li>
        <li>Upwork Job ID: 2009685381211068045</li>
        <li>Last Price Increase: 2026-01-09</li>
    </ul>
</details>

<details><summary>Issue Owner</summary>Current Issue Owner: @rojiphil</details>

**Comment by rojiphil** [comment]
@mallenexpensify Why not close this as suggested?

**Comment by rushatgabhane** [comment]
This issue was retested in **BFV cycle 544085 (Native Apps)** on 2026-04-28 and is **Not Fixed**.

### Screenshots / Videos
- https://platform.applause.com/services/links/v1/external/10a9e0e48572e6eba05297f338b478bb358b1e084777891b69d9c090832e8a83

### Retest notes
The issue is not fixed

The 'Licenses' text appears visually indistinguishable from surrounding plain text, with only a slight blue difference indicating its functionality. Users relying on non-color cues are unaware that the text is clickable, hindering navigation.

Issue verified on Xiaomi Redmi 11 Prime 5G - Android 13, Wi-Fi - Native
App Version: Version 9.3.61-4 and App Name: Expensify
Please see the attachments.

**Comment by rojiphil** [comment]
This issue was closed as per https://github.com/Expensify/App/issues/77244#issuecomment-3747033932

**Comment by rushatgabhane** [comment]
being discussed with QA
